enterprise/frontend/src/embedding-sdk/bin/generate-nextjs-compat.ts

```diff
@@ -0,0 +1,207 @@
+/* eslint-disable no-console */
+import fs from "fs";
+
+import path from "path";
+import prettier from "prettier";
+import { match } from "ts-pattern";
+
+type ComponentDefinition = {
+  mainComponent: string;
+  subComponents: string[];
+};
+
+// START OF CONFIGURATION
+// Note: this list needs to be updated when new components are added
+
+const COMPONENTS_TO_EXPORT: ComponentDefinition[] = [
+  // eslint-disable-next-line no-literal-metabase-strings -- cli tool
+
+  // MetabaseProvider is added manually because it needs to render children while loading
+  // we may have other components that need to render children while loading, in that case we can add a flag here
+  // { mainComponent: "MetabaseProvider", subComponents: [] },
+  { mainComponent: "StaticQuestion", subComponents: [] },
```

Comment:
The list of components is not completed btw, so not everything will work out of the box.
I  just wanted to check if the approach worked